lib/cookies.js

```diff
@@ -1,63 +1,49 @@
-/*
- * Simplecrawler - Cookie Jar Functionality
- * https://github.com/cgiffard/node-simplecrawler
- *
- * Copyright (c) 2011-2015, Christopher Giffard
- *
+/**
+ * @file simplecrawler's cookie jar module
  */
 
 var EventEmitter = require("events").EventEmitter,
     util = require("util");
 
-/*
-    Public: Constructor for the cookie jar.
-
-    Examples
-
-        var cookieJar = new CookieJar();
-
-    Returns the cookie jar object which has now been constructed.
-
-*/
-function CookieJar() {
-    var cookies = [];
-    this.__defineGetter__("cookies", function() {
-        return cookies;
-    });
-
-    // Run the EventEmitter constructor
+/**
+ * Creates a new cookie jar
+ * @class
+ */
+var CookieJar = function() {
     EventEmitter.call(this);
-}
-
-util.inherits(CookieJar, EventEmitter);
-
-/*
-    Public: Adds a new cookie to the jar, either by creating a new Cookie() object
-    from specific details such as name, value, etc., accepting a string from a
-    Set-Cookie header, or by passing in an existing Cookie() object.
-
-    name                -   The name of the cookie to add. Alternately, set-cookie
-                            header as string, or an existing cookie object.
-    value               -   The value of the cookie.
-    expiry              -   Expiry timestamp in milliseconds.
-    path                -   Limit cookie to path (defaults to "/")
-    domain              -   Limit cookie to domain
-    httponly            -   Boolean value specifying httponly
-    cb                  -   Optional callback.
-
-    Emits
 
-        addcookie       -   Emitted with new cookie object as an argument.
-
-    Examples
-
-        cookieJar.add("mycookie","myValue",Date.now(),"/","test.com",false);
+    /**
+     * The actual jar that holds the cookies
+     * @private
+     * @type {Array}
+     */
+    this.cookies = [];
```

Comment:
Was there a particular reason for the custom getter setup with the cookies array previously? Otherwise, this format worked right off the bat and makes it a bit easier to document and understand